src/cms/templates/analytics/_broken_links_widget.html

```diff
@@ -14,7 +14,36 @@
             {% trans "Details" %}
         </a>
     </div>
-    <div class="p-3 w-full">
-        <!-- TODO -->
+    <div class="mx-8">
```

Comment:
```suggestion
    <div class="mt-4">
```
I would rather add a small margin to the top instead of to the sides.